src/Dashboard.tsx

```diff
@@ -1,14 +1,10 @@
 import { useState } from "react";
 import Button from "./components/Button/Button";
 import Input from "./components/Input/Input";
+import { Todo } from "./components/TodoItemList/TodoItemList";
```

Comment:
You don't need two lines to import multiple things from the same file, condense this to a single line. Look up importing default and non-default imports in typescript.

Reply:
default imports go first. now i know this